CONTRIBUTING.md

```diff
@@ -366,6 +366,8 @@ make check-policies
 - Ensure the referenced specification version is up to date. Use
   [tooling to update the spec](#updating-the-referenced-specification-version)
   if needed.
+- Run [opentelemetry.io workflow](https://github.com/open-telemetry/opentelemetry.io/actions/workflows/build-dev.yml)
+  against `semantic-conventions` submodule as a smoke-test for docs. Fix broken links, if any.
```

Comment:
will this run against the (existing) semantic convention repo tag that's pinned currently in the website repo?

https://github.com/open-telemetry/opentelemetry.io/blob/a025c25aaf1aef653caa34e49e8714472cbeddbd/.gitmodules#L19-L22

Reply:
yep, it will run against any ref that you provide

<img width="989" alt="image" src="https://github.com/user-attachments/assets/4bb5fd7a-63db-4194-bf1d-a5e8c87927bf">

Reply:
oh, nice!